prow/scripts/cluster-integration/generate-self-signed-cert.sh

```diff
@@ -22,8 +22,8 @@ openssl req -x509 -nodes -days 5 -newkey rsa:4069 \
                  -keyout "${KEY_PATH}" \
                  -out "${CERT_PATH}"
 
-TLS_CERT=$(< /cert.pem base64 | tr -d '\n')
-TLS_KEY=$(< /key.pem base64 | tr -d '\n')
+TLS_CERT=$(base64 "${SCRIPTS_DIR}"/cert.pem | tr -d '\n')
```

Comment:
you already have variables that point to this files

Reply:
Fixed